langserver/handler_common.go

```diff
@@ -27,10 +26,10 @@ func (h *HandlerCommon) Reset(rootURI string) error {
 	if h.shutdown {
 		return errors.New("unable to reset a server that is shutting down")
 	}
-	if !strings.HasPrefix(rootURI, "file:///") {
+	if !isUri(rootURI) {
```

Comment:
there was a single place where we have checked for double slash, but according to comments it's safe to use the same triple-slash-check as we do in other places, because we need to test `URI or not`

```		// HACK: RootPath is not a URI, but historically we treated it
		// as such. Convert it to a file URI
		if !strings.HasPrefix(params.RootPath, "file://") {
			params.RootPath = "file://" + params.RootPath
		}
```